packages/utilities/psammead-styles/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "psammead-styles",
+  "version": "0.1.0",
+  "description": "A collection of BBC colours and fonts.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BBC-News/psammead.git"
```

Comment:
It'd be good to link to the exact location of this package:
`"url": "https://github.com/BBC-News/psammead/tree/latest/packages/utilities/psammead-styles"`